docs/Brew-Livecheck.md

```diff
@@ -75,63 +75,81 @@ We sometimes make this more explicit to exclude unwanted matches. URLs with a pr
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
-  end
+livecheck do
+  url :stable
+  regex(/^v?(\d+(?:\.\d+)+)$/i)
+end
 ```
 
 When the `stable` URL uses the `Git` strategy, the regex above will only match tags like `1.2`/`v1.2`, etc.
 
 If tags include the software name as a prefix (e.g. `example-1.2.3`), it's easy to modify the regex accordingly: `/^example[._-]v?(\d+(?:\.\d+)+)$/i`
 
-### `PageMatch` `strategy` block
+### `strategy` blocks
+
+If the upstream version format needs to be manipulated to match the formula/cask format, a `strategy` block can be used instead of a `regex`. 
+
+#### `PageMatch` `strategy` block
+
+In the example below, we're converting a date format like `2020-01-01` into `20200101`.
 
 ```ruby
-  livecheck do
-    url :homepage
-    regex(/href=.*?example[._-]v?(\d{4}-\d{2}-\d{2})\.t/i)
-    strategy :page_match do |page, regex|
-      page.scan(regex).map { |match| match&.first&.gsub(/\D/, "") }
-    end
+livecheck do
+  url :homepage
+  strategy :page_match do |page|
+    page.scan(/href=.*?example[._-]v?(\d{4}-\d{2}-\d{2})\.t/i)
+        .map { |match| match&.first&.gsub(/\D/, "") }
   end
+end
 ```
 
-When necessary, a `strategy` block allows us to have greater flexibility in how upstream version information is matched and processed. Currently, they're only used when the upstream version format needs to be manipulated to match the formula/cask format. In the example above, we're converting a date format like `2020-01-01` into `20200101`.
-
 The `PageMatch` `strategy` block style seen here also applies to any strategy that uses `PageMatch` internally.
 
-### `Git` `strategy` block
+#### `Git` `strategy` block
+
+A `strategy` block for `Git` is a bit different, as the block receives an array of tag strings instead of a page content string. Similar to the `PageMatch` example, this is converting tags with a date format like `2020-01-01` into `20200101`.
 
 ```ruby
-  livecheck do
-    url :stable
-    regex(/^(\d{4}-\d{2}-\d{2})$/i)
-    strategy :git do |tags, regex|
-      tags.map { |tag| tag[regex, 1]&.gsub(/\D/, "") }.compact
-    end
+livecheck do
+  url :stable
+  strategy :git do |tags|
+    tags.map { |tag| tag[/^(\d{4}-\d{2}-\d{2})$/i, 1]&.gsub(/\D/, "") }.compact
   end
+end
 ```
 
-A `strategy` block for `Git` is a bit different, as the block receives an array of tag strings instead of a page content string. Similar to the `PageMatch` example, this is converting tags with a date format like `2020-01-01` into `20200101`.
+#### `Sparkle` `strategy` block
 
-### Skip
+A `strategy` block for `Sparkle` receives an `item` which has methods for the `short_version`, `version`, `url` and `title`.
+
+The default pattern for the `Sparkle` strategy is `"#{item.short_version},#{item.version}"` if both are set. In the example below, the `url` also includes a download ID which is needed:
 
 ```ruby
-  livecheck do
-    skip "No version information available"
+livecheck do
+  url "https://updates.devmate.com/com.coppertino.Vox.xml"
```

Comment:
```suggestion
  url "https://www.example.com/example.xml"
```

I tend to use example URLs in the documentation, to avoid any potential issues that may arise from using a live URL (not from example.com or a domain under our control).